packages/charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -20,6 +20,8 @@ interface IDonutChartState {
   activeLegend?: string;
   color?: string | undefined;
   isLegendSelected?: boolean;
+  xCalloutValue: string;
+  yCalloutValue: string;
```

Comment:
question for you can we make this part of IDonutChartProps, this way we don't have to make this a breaking change

Reply:
In IDonutChartProps, these values are optional. In IDonutChartState I am using these are as mandatory in respective components.